src/demos/toggles.js

```diff
@@ -0,0 +1,43 @@
+(function() {
```

Comment:
nitpick: Should we have this in typescript? :)

Reply:
This folder doesn't get built automatically right now I don't believe. We'd have to introduce a new step in the build to do that. personally, it doesn't seem worth it right at this moment, but I think it may be a good step in the future. You ok with holding off for now?

Reply:
Yep. hold off!

Reply:
all yours bro